bsp/src/org/jetbrains/bsp/data/dataObjects.scala

```diff
@@ -53,8 +56,28 @@ object ScalaSdkData {
   * @param targetIds target ids mapped to module
   */
 @SerialVersionUID(4)
-case class BspMetadata @PropertyMapping(Array("targetIds"))(@NotNull targetIds: util.List[URI])
+case class BspMetadata @PropertyMapping(Array("targetIds", "scalaTestClasses"))(
+                                                                                 @NotNull targetIds: util.List[URI],
+                                                                                 @NotNull testClasses: util.List[String])
 object BspMetadata {
+
+  import com.intellij.openapi.externalSystem.util.{ExternalSystemApiUtil => ES}
+
   val Key: Key[BspMetadata] = datakey(classOf[BspMetadata])
+
+  def get(project: Project, module: Module): Option[BspMetadata] = {
```

Comment:
Most probably this method does not fit here. I tried to make use of 'BspMetadataService', however did not get anything sensible

Reply:
looks sensible